baseplate/observers/metrics_tagged.py

```diff
@@ -0,0 +1,197 @@
+from enum import Enum
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from baseplate import _ExcInfo
+from baseplate import BaseplateObserver
+from baseplate import LocalSpan
+from baseplate import RequestContext
+from baseplate import Span
+from baseplate import SpanObserver
+from baseplate.lib import config
+from baseplate.lib import metrics
+from baseplate.observers.timeout import ServerTimeout
+
+
+class Errors(Enum):
+    TIMED_OUT = 1
+    EXCEPTION = 2
+
+
+error_messages = {
+    Errors.TIMED_OUT: "timed_out",
+    Errors.EXCEPTION: "internal_server_error",
+}
```

Comment:
Note, you can define enums as:

```python
class Errors(Enum):
    TIMED_OUT = "timed_out"
    EXCEPTION = "internal_server_error"
```

and use `err_enum.value` to get the string value.